language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/SetterCodeAction.java

```diff
@@ -0,0 +1,70 @@
+package org.ballerinalang.langserver.codeaction.providers;
```

Comment:
Missing license header

Reply:
Updated in 85613cc5bfa90a74b38fb3e48f237c79dde35307